### ISSUE TITLE
Version string not linked into release 2.1.1

### ISSUE THREAD

**Issue by jecassis** [algorand/indexer#191]
Here is the output:
```
$ curl -sSLO https://github.com/algorand/indexer/releases/download/2.1.1/algorand-indexer_linux_amd64_2.1.1.tar.bz2
$ tar -xjf algorand-indexer_linux_amd64_2.1.1.tar.bz2
$ ./algorand-indexer_linux_amd64_2.1.1/algorand-indexer -v
(unknown version) compiled at  from git hash
```

Similarly the Debian package install also exhibits the missing version string:
```
$ algorand-indexer -v
(unknown version) compiled at  from git hash
```

**Comment by brianolson** [comment]
`misc/release.py` needs to be updated with the version tricks that `Makefile` is doing

**Comment by jecassis** [comment]
Closing, fixed in 2.2.1.
